### PR TITLE
Update docker images version

### PR DESCRIPTION
**Pull request by DanielFran**
---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/common/templates/sonar-project.properties.ejs [hunk 1]
```diff
@@ -2,6 +2,10 @@ sonar.projectKey=<%= baseName %>
 sonar.projectName=<%= baseName %> generated by jhipster
 sonar.projectVersion=1.0
 
+# After changing password in SonarQube change this also here
+#sonar.login=admin
+#sonar.password=admin
```
Comment (kaidohallik) [inline generators/common/templates/sonar-project.properties.ejs:7]:
```suggestion
sonar.login=admin
sonar.password=admin
```

This is working out of the box, so I prefer to uncomment this.
If user logs in to web interface then user is forced to change password. After password change new password is needed to put also here.

**Review by kaidohallik — APPROVED** [review]

**Comment by kaidohallik** [comment]
I was wrong when guessing that token in GitHub allows still to connect to SonarCloud. Making PR shortly to comment those `sonar.login` and `sonar.password` properties.

**Comment by kaidohallik** [comment]
In #13293 I commented out `sonar.login` and `sonar.password` to make SonarCloud authentication to work again.
Now found solution to make SonarQube locally to work without authentication, add this to `sonar.yml` docker config:
```
    environment:
      - sonar.forceAuthentication=false
```
Should we change those commented out authentication instructions in `sonar-project.properties` to this authentication turn off in Docker configuration for SonarQube?

**Comment by DanielFran** [comment]
@kaidohallik Probably, yes!

**Comment by kaidohallik** [comment]
I made #13302 for turning off SonarQube authentication.

**Pull request merged**